src/pages/faq/faq.mdx

```diff
@@ -0,0 +1,133 @@
+---
+name: FAQ
+route: /faq
+---
+
+# FAQ
+
+## How can I listen to form changes in the standalone component?
+
+When using JSON Forms within your react app, you probably want to access the current form data. 
```

Comment:
When using JSON Forms within your react app, at some point you'll need to access the current form data.